src/components/reach-customer-sec.html

```diff
@@ -0,0 +1,17 @@
+<section class="reach-customer-sec">
+  <div class="container">
+    <div class="header-part">
+      <h1 class="header">Reach More Customers</h1>
+      <div class="header-links">
+        <button href="javascript:void(0)" class="learn-how">learn-how</button>
+      </div>
+    </div>
+    <figure>
+      <img
+        src="../assets/images/secondSection/VIDEO.png"
+        alt="video"
+        class="video"
+      />
```

Comment:
Is it a video? If yes, then this should be a <video> and the image you are using is a thumb of that video